MapboxNavigationUI/NavigationMapView.swift

```diff
@@ -54,15 +57,18 @@ open class NavigationMapView: MGLMapView {
         }
     }
     
-    public func shape(describing route: Route) -> MGLShape? {
+    open func shape(describing route: Route) -> MGLShape? {
         guard var coordinates = route.coordinates else {
             return nil
         }
         
         return MGLPolylineFeature(coordinates: &coordinates, count: route.coordinateCount)
     }
     
-    public func routeStyleLayer(identifier: String, source: MGLSource) -> MGLStyleLayer {
+    /**
+     Function for overriding the default route line style.
```

Comment:
These are methods, not functions. (A function stands alone, apart from any class.)